config/samples/nodeobservability_v1alpha2_nodeobservability.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
   name: cluster
 spec:
   nodeSelector:
-    node-role.kubernetes.io/worker: ""
+    kubernetes.io/hostname: worker-a
```

Comment:
I'm guessing this was unintentional? Since worker nodes don't actually get this label. 
```bash
ip-10-0-171-177.ec2.internal   Ready    worker   3h30m   v1.24.0+1ff0731   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/instance-type=m6i.xlarge,beta.kubernetes.io/os=linux,failure-domain.beta.kubernetes.io/region=us-east-1,failure-domain.beta.kubernetes.io/zone=us-east-1c,kubernetes.io/arch=amd64,kubernetes.io/hostname=ip-10-0-171-177.ec2.internal,kubernetes.io/os=linux,node-role.kubernetes.io/worker=,node.kubernetes.io/instance-type=m6i.xlarge,node.openshift.io/os_id=rhcos,topology.ebs.csi.aws.com/zone=us-east-1c,topology.kubernetes.io/region=us-east-1,topology.kubernetes.io/zone=us-east-1c

```

Reply:
The value of the label is irrelevant, it's just an example. I didn't want it to look as close as possible to the real values. I just wanted to show a sample which would be more granular than "all workers".
However on GCP the hostname label does contain `worker-a` substring.

Reply:
Got it. I generally use the samples during testing (I guess this is going to be an extra step). But the expected key on GCP is not `kubernetes.io/hostname:`. Right?

Reply:
Yes, the expected key is `kubernetes.io/hostname`, it's present on all nodes on GCP and AWS at least.